skore/src/skore/sklearn/_estimator/metrics_accessor.py

```diff
@@ -2024,10 +2025,15 @@ def confusion_matrix(
             pos_label=None,
         )
 
-        return ConfusionMatrixDisplay.from_predictions(
-            y_true=y,
-            y_pred=y_pred,
-            sample_weight=sample_weight,
+        cm = sklearn_confusion_matrix(
```

Comment:
To follow the other display api, we should implement a `_compute_data_for_display` class method in the display. Basically, it will replace the previous `from_predictions`.